test/support/spawnServer.ts

```diff
@@ -93,14 +96,12 @@ Using Service Worker build:
       // to be registered at the website's root.
       'Service-Worker-Allowed': '/',
     },
-    after(app, server, compiler) {
+    before(app, server, compiler) {
+      options?.withRoutes?.(app, server, compiler)
```

Comment:
Per [WDS documentation](https://webpack.js.org/configuration/dev-server/#devserverbefore), `before` is the right option to attach custom route handlers.